velox/type/Variant.cpp

```diff
@@ -20,6 +20,8 @@
 #include "velox/common/encode/Base64.h"
 #include "velox/type/DecimalUtil.h"
 #include "velox/type/FloatingPointUtil.h"
+#include "velox/common/base/BitUtil.h"
+#include "velox/vector/BaseVector.h"
```

Comment:
Can we avoid introduce dependency from "type" to "vector"?

Reply:
ack